scripts/release.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+ set -eu
+set -o pipefail
+
+ echo -n "Build timestamp: "
+TIMESTAMP=$(date +%s)
+echo $TIMESTAMP
+
+ echo -n "Git commit: "
+if [[ "$(git diff --stat)" != '' ]] || [[ -n "$(git status -s)" ]]; then
+  COMMIT=$(git rev-parse HEAD)-dirty
+  RELEASE=false
+else
+  COMMIT=$(git rev-parse HEAD)
+  RELEASE=true
+fi
+echo $COMMIT
+
+ echo -n "Tagged version: "
+VERSION=$(git describe --tags --match "v[0-9]*.[0-9]*.[0-9]*")
```

Comment:
this is an even simpler semantic version matcher. not a regex, but i think we can safely be more strict in the code because of this